crates/wash-lib/src/cli/start.rs

```diff
@@ -72,9 +73,111 @@ pub struct StartComponentCommand {
     /// If this flag is omitted, the timeout will be adjusted to 5 seconds to account for component download times
     #[clap(long = "skip-wait")]
     pub skip_wait: bool,
+
+    /// Digest to verify artifact against (if OCI URL is provided for <target>)
+    #[clap(short = 'd', long = "digest")]
+    pub digest: Option<String>,
+
+    /// Allow latest artifact tags (if OCI URL is provided for <target>)
+    #[clap(long = "allow-latest")]
+    pub allow_latest: bool,
+
+    /// OCI username, if omitted anonymous authentication will be used
+    #[clap(long = "user", env = "WASH_REG_USER", hide_env_values = true)]
+    pub user: Option<String>,
+
+    /// OCI password, if omitted anonymous authentication will be used
+    #[clap(long = "password", env = "WASH_REG_PASSWORD", hide_env_values = true)]
+    pub password: Option<String>,
+
+    /// Allow insecure (HTTP) registry connections
+    #[clap(long = "insecure")]
+    pub insecure: bool,
+
+    /// skip the local OCI cache and pull the artifact from the registry to inspect
+    #[clap(long = "no-cache")]
+    pub no_cache: bool,
 }
 
-pub async fn handle_start_component(cmd: StartComponentCommand) -> Result<CommandOutput> {
+pub async fn handle_start_component(cmd: StartCommand) -> Result<CommandOutput> {
```

Comment:
Let's call this `handle_start` and have `handle_start_component` still be the way to start what was used to be called an "actor." We've moved away from using the actor terminology and just call them "components" now, since that is what they are